lib/CreditNote.php

```diff
@@ -78,6 +82,22 @@ public static function preview($params = null, $opts = null)
         return $obj;
     }
 
+    /**
+     * @param array|null $params
+     * @param array|string|null $opts
+     *
+     * @throws \Stripe\Exception\ApiErrorException if the request fails
+     *
+     * @return CreditNote The previewed credit note.
+     */
+    public function previewLines($params = null, $opts = null)
+    {
+        $url = $this->instanceUrl() . '/lines';
```

Comment:
[blocker]

This is incorrect because it should be static and it doesn't have an instanceUrl because it's not on an existing credit note. Your think works if you have called preview first but IMO this is the wrong abstraction/name

We should blacklist this method for now instead of trying to fix it